src/Controls/src/Core/Layout.cs

```diff
@@ -263,10 +260,10 @@ protected void UpdateChildrenLayout()
 			if (!ShouldLayoutChildren())
 				return;
 
-			var oldBounds = new Rect[LogicalChildrenInternal.Count];
+			var oldBounds = new Rect[InternalChildren.Count];
```

Comment:
Changing all uses of `LogicalChildrenInternal` to `InternalChildren` shouldn't change any behavior. The two were already used interchangeabily inside the code.